cmd/artifact/Makefile

```diff
@@ -1,10 +1,12 @@
+ARTIFACT_ID=master-`date +"%Y%m%d"`-`date +"%H%M"`
```

Comment:
Would be nice to move these Make targets to be available from the main make file, but i guess that's a separate PR.

Reply:
Yeah, agree. That would be great :)